app/newsfeed/server/deinitialize.js

```diff
@@ -2,13 +2,15 @@
 //
 // import { settings } from '../../settings';
 import { deleteRoom } from '../../lib/server/functions';
-import { Rooms } from '../../models';
+import { Rooms, Users } from '../../models';
 
 function deinitializeNewsfeed() {
 	const roomList = Rooms.findByType('n').fetch();
 	roomList.forEach((room) => {
 		deleteRoom(room._id);
 	});
+	Users.update({}, { $unset: { followers: 1 } }, { multi: true });
+	Users.update({}, { $unset: { following: 1 } }, { multi: true });
```

Comment:
Same thing here, do we want all this data to be disabled, or deleted permanently?

Reply:
I suggest that it should be deleted as the administrator would want to delete all trace of newsfeed

Reply:
seems fine as a first implementation